gallery/src/styles/NxTile/NxTileWithHorizontalRuleExample.html

```diff
@@ -1,26 +0,0 @@
-<!--
```

Comment:
why remove this file? unused?

Reply:
> why remove this file? unused?

Yeah, we removed that example last week and didn't remove the file, so it's just a bit of housekeeping